vxquery-core/src/main/java/org/apache/vxquery/functions/builtin-functions.xml

```diff
@@ -128,6 +128,28 @@
         <!-- Collection operator is added during the rewrite rules phase.  -->
     </function>
     
+    <!-- fn:index-from-collection($arg  as xs:string?, $indexFolder  as xs:string?) as  node()* -->
+    <function name="fn:index-from-collection">
+        <param name="collectionFolder" type="xs:string?"/>
```

Comment:
Maybe the user facing name should be something like: build-index-from-collection